### ISSUE TITLE
XEPs and RFCs support page with version

### ISSUE THREAD

**Issue by Neustradamus** [tigase/tigase-server#12]
It is possible to have a XEPs and RFCs support page with version (XEP-XXXX v1.2)?

Of course, it is for all projects/repositories.

Examples:
- https://metronome.im/supported-xeps-list
- https://dev.gajim.org/gajim/gajim/wikis/help/gajimxepsupport
- https://github.com/jappix/jappix/blob/master/PROTOCOL.md

**Comment by woj-tek** [comment]
We *do* have list with supported RFCs and XEPs: https://github.com/tigase/tigase-server/blob/master/src/main/asciidoc/admin/text/admin/FeaturesXMPP.asciidoc

Though, versions are missing and we could add them.

**Comment by arturhefczyc** [comment]
Yes, I think information about spec version supported might be useful as it indicates whether we support specific feature added to the spec at later time.

**Comment by ecore78** [comment]
want tigase support xep-0369(MIX),do you have the plan?

**Comment by woj-tek** [comment]
Yes, MIX is on our roadmap, but we can't provide ETA for now.

**Comment by arturhefczyc** [comment]
We are actively working on MIX and we expect to have first working version for tests at FOSDEM this year.
